docs/docs/guides/integrate/identity-providers/introduction.md

```diff
@@ -0,0 +1,182 @@
+  - **Create a new account**: Choosing this option creates a new ZITADEL account linked to the external IdP.
+  - **Autolinking with an existing local account**: Users have the option to link their new external identity to an existing local account in ZITADEL, enabling future logins with either their local account or the external IdP.
+
+
+- **Profile pre-filling from external IdP**: ZITADEL uses information from the external IdP to pre-fill the user's profile, simplifying the account creation or linking process.
+
+
+- **User option to update profile**: Users can review and, if necessary, update their pre-filled profile information.
+
+
+- **Session creation and access granting**: After the account is created or linked and the profile is set, the application grants access by creating a session for the user based on their authenticated identity.
+
+
+## Setting up external identity providers in ZITADEL
+
+
+In ZITADEL, you have the flexibility to link an external Identity Provider (IdP) to your entire instance, making it the default option for all organizations within your instance, or to connect it exclusively to a specific organization. This setup allows organization members to leverage similar capabilities in self-service if permitted.
+
+
+### Adjusting the custom login policy
+
+
+The login policy can be set as a default at the instance level and can be customized for each organization. The configuration process varies slightly depending on your focus:
+
+
+- **For instance-wide settings**, navigate to: `$YOUR-DOMAIN/ui/console/instance?id=general`
+- **For specific organization settings**, select the organization from the menu and visit: `$YOUR-DOMAIN/ui/console/org-settings?id=login`
+
+
+Once in the settings:
+- Access the **Login Behavior and Security** section to modify your login policy. Here, ensure you enable the option for **External IDP Allowed**.
+
+
+![Allow External IDP](/img/guides/zitadel_allow_external_idp.png)
+
+
+
+### Configuring IdP Providers
+
+
+Access the settings page of your instance or the specific organization and select **Identity Providers**.
+
+
+The ZITADEL Console will display a list of all the IdPs you've configured, along with available provider templates. Selecting any listed IdP will guide you through the process of configuring that specific Identity Provider.
+
+![Identity Provider Overview](/img/guides/zitadel_identity_provider_overview.png)
+
+
+## Available templates
+
+In the guides below, you'll learn how to configure and set up your preferred external Identity Provider (IdP) in ZITADEL.
+
+- [Google](./google)
+- [Entra ID (OIDC)](./azure-ad-oidc)
+- [Entra ID SAML](./azure-ad-saml)
+- [GitHub](./github)
+- [GitLab](./gitlab)
+- [Apple](./apple)
+- [LDAP](./ldap) 
+- [Local OpenLDAP](./openldap.mdx)
+- [OKTA generic OIDC](./okta-oidc)
+- [OKTA SAML](./okta-saml)
+- [Keycloak generic OIDC](./keycloak)
+- [MockSAML](./mocksaml)
+- [JWT IdP](./jwt_idp)
```

Comment:
These are not all available templates, some of them are based on the generic tempaltes. We just provide guides on how to configure them

Reply:
Changed.